app/tests/archives_tests/test_views.py

```diff
@@ -999,3 +1001,183 @@ def test_archive_item_bulk_delete_permissions(client):
         list(response.context["form"].fields["civ_sets_to_delete"].queryset)
         == []
     )
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "distractor_generator",
```

Comment:
What is a distractor?

Reply:
The general setup of the test is to ensure that a similar-looking set of inputs to a Job are NOT searched upon. Hence I minted them 'Distractors'. I could abstract the creation of the non-distractors into the params definition, but that seemed to me to be overcomplicating things.

Reply:
Maybe name it differently? That term is non-standard and non-obvious. Better to choose something that tells the future reader exactly what is going on.

Reply:
Ok, I've updated it to make more sense: I hope.